src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -135,47 +148,57 @@ export const RegistrationFormActions = ({
           onClick={toggleSupportModal}>
           {t('my_page.messages.get_curator_support')}
         </Button>
-        {!isLastTab ? (
-          <Box
-            sx={{
-              gridArea: 'save-button',
-              display: 'flex',
-              justifyContent: 'end',
-              alignItems: 'center',
-            }}>
+
+        <Box
+          sx={{
+            gridArea: 'action-buttons',
+            display: 'flex',
+            justifyContent: 'end',
+            alignItems: 'center',
+          }}>
+          <Button
+            data-testid={dataTestId.registrationWizard.formActions.cancelEditButton}
+            onClick={cancelEdit}
+            color="primary"
+            sx={{ width: 'fit-content', justifySelf: 'center' }}>
```

Comment:
Trengs dette da? Siden flexen har `alignItems` og `justifyContent` virker det som dette bare gjentar stylingen den allerede har fra container 🤔 
```suggestion
            >
```